### PR TITLE
Adds dropbox lunch event

### PR DESCRIPTION
**Pull request by bdamerac**

### CODE REVIEW

File: content/W2017/W17-event-dropbox-lunch.md [hunk 1]
```diff
@@ -0,0 +1,23 @@
+Title: Dropbox Lunch Time
+Date: 2017-02-24
+Category: Events
+Tags: social
+Slug: dropbox-lunch-w17
+Author: Bhargavi Dameracharla
+Summary: Chat with dropbox recruiters and engineers!
+
+WiCS is hosting an event with Dropbox so that you can
+speak to the recruiters and engineers at dropbox, and
```
Comment (evykassirer) [inline content/W2017/W17-event-dropbox-lunch.md]: capitalize dropbox here

File: content/W2017/W17-event-dropbox-lunch.md [hunk 2]
```diff
@@ -0,0 +1,23 @@
...
+process as well! So come with questions and lots of
+appetite!
+
+Note: This event is [women-only]({filename}/pages/faq.md).
+
+## Event Details ##
+
++ **Who:** [Women-only]({filename}/pages/faq.md)
++ **What:** Dropbox Lunch
++ **Where:** William's Coffee Pub, 170 University Ave W
```
Comment (annalorimer) [inline content/W2017/W17-event-dropbox-lunch.md]: It's just William's Coffee

File: content/W2017/W17-event-dropbox-lunch.md [hunk 3]
```diff
@@ -0,0 +1,23 @@
...
+Slug: dropbox-lunch-w17
+Author: Bhargavi Dameracharla
+Summary: Chat with Dropbox recruiters and engineers!
+
+WiCS is hosting an event with Dropbox so that you can
+speak to the recruiters and engineers at Dropbox, and
+ask them any questions you have about Dropbox. This is
+a chance for you to get familiar with their hiring
+process as well! So come with questions and lots of
+appetite!
```
Comment (annalorimer) [inline content/W2017/W17-event-dropbox-lunch.md:14]: Should mention that it's drop in. Also the word Dropbox is repeated too often 😝

**Comment by annalorimer** [comment]
@Bhargavi1997 how's this going?

**Pull request merged**